ui/analyse/css/_training.scss

```diff
@@ -9,6 +9,12 @@
     flex-flow: column;
     justify-content: center;
 
+    @include mq-is-col1 {
+      &.view {
```

Comment:
that only works when you ask to see the solution. Not when you find it yourself.